### ISSUE TITLE
Suggestion:

### ISSUE THREAD

**Issue by denistsoi** [thenewboston-developers/ui-js#9]

**Comment by angle943** [comment]
yeah we have storybook for our `ui` library. But it'll be good to have here too

**Comment by denistsoi** [comment]
@angle943  - I can't seem to add this repo (as i'm not included as a member in thenewbostons-developers)
I forked the repo and currently hosted on netlify 
https://6070036ac9b9656e95881fad--tnbc-developers-ui-js.netlify.app/?path=/story/button--default

**Comment by angle943** [comment]
ah nice, thanks! let's talk about this in private chat, message me on discord
